app/controllers/hub/metrics_controller.rb

```diff
@@ -6,20 +6,38 @@ class MetricsController < ApplicationController
     load_and_authorize_resource :vita_partner, parent: false
 
     def index
+      # @vita_partners = @vita_partners.order(name: :asc)
       # Run SLA queries and cache the result for 10 minutes.
       # The view will only display values for vita partners the user has access to view.
       data = Rails.cache.fetch("metrics/sla_breaches/attention_needed", expires_in: 10.minutes) do
         sla_service = SLABreachService.new
-        { breach_threshold_date: sla_service.breach_threshold_date,
+        attention_needed_breaches = sla_service.attention_needed_breaches
```

Comment:
This is kind of a quick-and-dirty way to expose the new query results into the view. A subsequent PR removes a lot of logic from this controller.